go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/prometheus-operator/prometheus-operator/pkg/client v0.56.3
 	github.com/prometheus/common v0.37.0
 	github.com/prometheus/prometheus v1.8.2-0.20220315145411-881111fec433
+	github.com/robfig/cron/v3 v3.0.1
```

Comment:
Confirm: does velero depend on it

Reply:
velero depend on github.com/robfig/cron v1.1.0 

they are two versions of the cron package in a project, it should be avoid.